src/pages/LoginAndRegister/GuestAccountClaim.vue

```diff
@@ -77,13 +85,17 @@ export default {
 	methods: {
 		claimGuestAccount() {
 			this.$v.$touch();
+
 			if (!this.$v.$invalid) {
 				const params = [
 					`firstName=${encodeURIComponent(this.firstName)}`,
 					`lastName=${encodeURIComponent(this.lastName)}`,
 					`state=${this.$route.query.state}`
 				].join('&');
+				this.$kvTrackEvent('Register', 'guest-account-registration-success');
```

Comment:
Because you're setting `window.location` on the next line, this tracking request might not go out before. the page is redirected away from. You should move the `window.location` line into the callback for $kvTrackEvent, like you did in the auth0 PR https://github.com/kiva/auth0/pull/225/files#diff-87867a89267cfde054fc5b173b4248ed9adcaa69154aa83233462eb3aed693c0R151